file-native/src/main/java/org/ballerinalang/stdlib/file/utils/FileConstants.java

```diff
@@ -48,18 +48,42 @@
      */
     public static final Module FILE_PACKAGE_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, PACKAGE_NAME,
                                                                 PACKAGE_VERSION);
-    static final String FILE_INFO_TYPE = "FileInfo";
+    public static final String METADATA = "MetaData";
 
     // File error type IDs
     public static final String INVALID_OPERATION_ERROR = "InvalidOperationError";
     public static final String PERMISSION_ERROR = "PermissionError";
     public static final String FILE_SYSTEM_ERROR = "FileSystemError";
     public static final String FILE_NOT_FOUND_ERROR = "FileNotFoundError";
+    public static final String NOT_LINK_ERROR = "NotLinkError";
+    public static final String IO_ERROR = "IOError";
+    public static final String SECURITY_ERROR = "SecurityError";
+    public static final String INVALID_PATH_ERROR = "InvalidPathError";
+    public static final String GENERIC_ERROR = "GenericError";
     static final String ERROR_DETAILS = "Detail";
     static final String ERROR_MESSAGE = "message";
 
     // System constant fields
-    public static final int DEFAULT_MAX_DEPTH = -1;
+    public static final int MAX_DEPTH = 1;
+
+    // Enum constants
+
+    public static final String REPLACE_EXISTING = "REPLACE_EXISTING";
+    public static final String COPY_ATTRIBUTES = "COPY_ATTRIBUTES";
+    public static final String NO_FOLLOW_LINKS = "NO_FOLLOW_LINKS";
+    public static final String EXISTS = "EXISTS";
+    public static final String IS_DIR = "IS_DIR";
+    public static final String IS_SYMLINK = "IS_SYMLINK";
+    public static final String READABLE = "READABLE";
+    public static final String WRITABLE = "WRITABLE";
+    public static final String RECURSIVE = "RECURSIVE";
```

Comment:
Shall we have inner classes to group these Constants such as
FileOption, MetaDataFields and so on?

Reply:
I just wanted to have all the constants in one place for readability. Let's not complicate it by adding inner classes ATM.

Reply:
They are supposed to be in one file, Within the constants file we have different inner classes, https://github.com/ballerina-platform/module-ballerina-sql/blob/master/sql-native/src/main/java/org/ballerinalang/sql/Constants.java#L75 This is especially readble for ballerina record fields